.github/workflows/integration_test.yml

```diff
@@ -15,26 +15,10 @@ jobs:
         with:
           go-version: 1.18
 
-      - name: install kernel headers
-        run: |
-          sudo apt-get install -y linux-headers-`uname -r`
-          sudo ls /usr/src/linux-headers-`uname -r`
-
-      - name: install bcc
-        run: |
-            wget https://github.com/sustainable-computing-io/kepler-ci-artifacts/releases/download/v0.25.0/bcc_v0.25.0.tar.gz
-            tar -zxvf bcc_v0.25.0.tar.gz
-            sudo dpkg -i libbcc_0.25.0-1_amd64.deb
-
-      - name: install kubectl
-        run: curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
-
-      - name: start local k8s cluster
-        run: make cluster-up
-        env:
-            CLUSTER_PROVIDER: ${{matrix.kube_provider}}
-            PROMETHEUS_ENABLE: "true"
-            GRAFANA_ENABLE: "false"
+      - name: use kepler action for kind cluster build
+        uses: sustainable-computing-io/KeplerK8SAction@main
```

Comment:
This is outside of the scope of this PR, but I would suggest renaming the action to `sustainable-computing-io/kepler-action` so that it's lowercase, kebab-case, and a bit simplified, to follow the naming pattern of other GitHub Actions e.g. see [release-drafter](https://github.com/marketplace/actions/release-drafter), [create-pull-request](https://github.com/marketplace/actions/create-pull-request), etc.

Reply:
I can open an issue in the action's repo if you agree!

Reply:
yep, please go ahead to open an issue in action's repo. we should update action 1st then here.

Reply:
@rootfs , if necessary, please rename our GHA repo.

Reply:
Great, opened an issue: https://github.com/sustainable-computing-io/KeplerK8SAction/issues/17